pakyow-presenter/lib/pakyow/presenter/form.rb

```diff
@@ -0,0 +1,65 @@
+module Pakyow
+  module Presenter
+    class Form
+
+      attr_reader :form_view, :context
+
+      def initialize(view, context)
+        @form_view = view
+        @context = context
+      end
+
+      def create(*binding_args, &block)
+        bind_data(binding_args, block) do |routes, view|
```

Comment:
Yeah, tough. Yielding routes here doesn't give us much, so maybe it can be moved. 

Off the top of my head, what about `bind_form_action` instead?